src/main/scala/org/broadinstitute/dsde/firecloud/model/Library.scala

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.dsde.firecloud.model
+
+import spray.json.JsObject
+
+/**
+  * Created by ahaessly on 11/2/16.
+  */
+
+  case class LibrarySearchParams(
+    searchTerm: String,
+    from: Int = 0,
+    size: Int = 10)
+
+  case class LibrarySearchResponse(
+    total: Int,
+    results: Array[JsObject])
```

Comment:
Also, I think the search response should echo the `LibrarySearchParams` it used back to the caller. This makes handling the response super easy, either from the UI or via script - the caller can read everything it needs from the response packet.